Apps/System/music.c

```diff
@@ -0,0 +1,461 @@
+/* music.c
+ *
+ * Music player
+ *
+ * RebbleOS
+ * 
+ * Author: Chris Multhaupt <chris.multhaupt@gmail.com>.
+ */
+
+#include "rebbleos.h"
+#include "music.h"
+#include "property_animation.h"
+
+#define RECORD_CENTER_X 56
+#define RECORD_CENTER_Y 57
+#define RECORD_CENTER GPoint(RECORD_CENTER_X, RECORD_CENTER_Y)
+#define RECORD_SWOOSH_DISTANCE 130
+#define RECORD_SWOOSH_SPEED 250
+#define ARM_OFFSET_X 17
+#define ARM_OFFSET_Y 77
+#define ARM_OFFSET GPoint(ARM_OFFSET_X, ARM_OFFSET_Y)
+#define ARM_OFFSET_ANGLE -31
+#define ARM_HOME_ANGLE 0
+#define ARM_START_ANGLE 18
+#define ARM_END_ANGLE 46
+#define ARM_SKIP_SPEED 200
+#define ARM_QUICK_SPEED 100
+#define SKIP_DIRECTION_PREV -1
+#define SKIP_DIRECTION_NEXT 1
+#define PROGRESS_PIXELS_MAX 105
+
+static Window *s_main_window;
```

Comment:
I've been naming static variables `_main_window`, rather than `s_main_window`.  I'm not sure what happens if there are other `_main_window`s (i.e., which one the debugger will pick up), though.